Fixed illustration of in-memory format of environment strings.

ai4rei opened the pull request:
Confirmed in both ANSI and UNICODE version of the function.

ai4rei committed:
Fixed illustration of in-memory format of environment strings.

PRMerger17 commented:
@ai4rei : Thanks for your contribution! The author(s) have been notified to review your proposed change.

Karl-Bridge-Microsoft committed:
Update nf-processenv-getenvironmentstrings.md

format info copied from https://docs.microsoft.com/windows/win32/procthread/environment-variables

ghost commented:
[![CLA assistant check](https://cla.opensource.microsoft.com/pull/badge/signed)](https://cla.opensource.microsoft.com/MicrosoftDocs/sdk-api?pullRequest=202) <br/>All CLA requirements met.

PRMerger9 commented:
@Karl-Bridge-Microsoft : Thanks for your contribution! The author(s) have been notified to review your proposed change.
